src/main/java/org/galatea/starter/service/IexService.java

```diff
@@ -86,9 +180,83 @@
    */
   public List<IexHistoricalPrices> getHistoricalPricesForSymbols(final String symbols) {
 
-    return iexHistoricalClient.getHistoricalPricesForSymbols(symbols);
+    return getHistoricalPricesForSymbols(symbols,"1m");
+  }
 
+  /**
+   * Get HistoricalPricesDB from the database and add it to the list.
+   *
+   * @param historicalPricesId object to searched in the database
+   * @param listIexHistoricalPrices list where object is to be added
+   */
+  public void getIexHistoricalPricesObjectFromDatabaseHelper(
+      final HistoricalPricesId historicalPricesId,
+      final List<IexHistoricalPrices> listIexHistoricalPrices) {
+
+    Optional<HistoricalPricesDB> listHistoricalPricesDB =
+        historicalPricesService.getHistoricalPriceFromDb(historicalPricesId);
+
+    HistoricalPricesDB objHistoricalPricesDB = listHistoricalPricesDB.get();
+
+    IexHistoricalPrices objIexHistoricalPrices = IexHistoricalPrices.builder().build();
```

Comment:
You have a builder, please look into using it instead of using all these setters